edit/buildozer.go

```diff
@@ -348,6 +348,31 @@ func cmdRemove(opts *Options, env CmdEnvironment) (*build.File, error) {
 	return nil, nil
 }
 
+func cmdRemoveComment(opts *Options, env CmdEnvironment) (*build.File, error) {
+	switch len(env.Args) {
+	case 0: // Remove comment attached to rule
+		env.Rule.Call.Comments.Before = nil
+		env.Rule.Call.Comments.Suffix = nil
```

Comment:
A rule (and in theory any node in the AST) may have an after-comment, please nullify `Rule.Call.Comments.After` too.

    my_rule(
        name = "foo" ,
    )
    # this comment is attached to the rule above